src/EFCore.PG/Extensions/NpgsqlIndexBuilderExtensions.cs

```diff
@@ -186,6 +186,63 @@ public static class NpgsqlIndexBuilderExtensions
             return indexBuilder.CanSetAnnotation(NpgsqlAnnotationNames.IndexOperators, operators, fromDataAnnotation);
         }
 
+        /// <summary>
+        /// The PostgreSQL columns transformation to 'to_tsvector()' to be used.
+        /// </summary>
+        /// <param name="indexBuilder">The builder for the index being configured.</param>
+        /// <param name="configName">to_tsvector() function config name.</param>
+        /// <returns></returns>
+        public static IndexBuilder UseToTsVector(
```

Comment:
Move these methods out of the Operators region and into their own region named "ToTsVector"